src/components/DeleteComment.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react"
+import { deleteComment } from "../../api"
+
+function DeleteComment({comment}) {
+    const [isDeleted, setIsDeleted] = useState(false)
+    const [isNotDeleted, setIsNotDeleted] = useState(false)
+    const [isDeleting, setIsDeleting] =useState(false)
+
+    //// Temp Logged in User ////
+    const username = "cooljmessy"
+
+    function handleClick() {
+        setIsDeleting(true)
+        deleteComment(comment.comment_id)
+        .then(() => {
+            setIsDeleting(false)
+            setIsDeleted(true)
+        })
+        .catch(() => {
+            setIsNotDeleted(true)
+            setIsDeleting(false)
+            setTimeout(() => {
+                setIsNotDeleted(false)
+            }, 3000);
+        })
+    }
+
+    return (
+        <>
+            {isDeleting ? <p className="deleting">deleting...</p> : null}
```

Comment:
deleting functionality works well but something to consider is that I'm able to delete anyone's comments and not just the ones I leave (although _I'd_ love to do this in real life, it may be considered a problem by others 😂 )